modules/new_profile/php/new_profile.class.inc

```diff
@@ -6,25 +6,27 @@
  *
  * @category Main
  * @package  Loris
- * @author   Ted Strauss <ted.strauss@mcgill.ca>
+ * @author   Shen Wang <shen.wang2@mcgill.ca>
  * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
  * @link     https://github.com/aces/Loris
  */
 namespace LORIS\new_profile;
-
 /**
  * New_profile
  *
- * PHP Version 5
+ * PHP Version 7
  *
  * @category Main
  * @package  Loris
- * @author   Ted Strauss <ted.strauss@mcgill.ca>
+ * @author   Shen Wang <shen.wang2@mcgill.ca>
  * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
  * @link     https://github.com/aces/Loris
  */
-class New_Profile extends \NDB_Form
+class New_Profile extends \NDB_Menu_Filter
```

Comment:
Why does this become a NDB_Menu_Filter? I don't think there's any SQL table on this page. From what I can tell you're just trying to avoid the _process function of NDB_Form, in which case it should be an \NDB_Page.